how-to-guides/e2e-tracking/scripts/e2e_tracking.py

```diff
@@ -49,27 +48,36 @@
 # To prevent `RuntimeError: main thread is not in main loop` error
 matplotlib.use("Agg")
 
-# ===== Track model training ===== #
+# %% ===== Track model training ===== #
 # 1. Use Optuna to train multiple scikit-learn regression models
 # 2. Leverage Neptune's scikit-learn and Optuna integrations to automatically log metadata and metrics
-#    to Neptune for easy run comparison, while also using Neptune's model registry to track models.
+#    to Neptune for easy run comparison
+# 3. Use Neptune's model registry to track models
 
-## Prepare the dataset ##
+# %%# Prepare the dataset ##
 
 data, target = fetch_california_housing(return_X_y=True)
 X_train, X_test, y_train, y_test = train_test_split(data, target, test_size=0.25)
 
-### Initialize a Neptune project
-# This step assumes that your Neptune API token is set as an environment variable
-# **Note:** The "common/e2e-tracking" project used here is read-only.
-# To log to your own project, just update the `NEPTUNE_PROJECT` environment variable.
+# %%## Initialize the Neptune project
+# To connect to the Neptune app, you need to tell Neptune who you are (`api_token`) and where to send the data (`project`).
+# By default, this script You can use the default code below log to the public project `common/e2e-tracking` as an anonymous user.
```

Comment:
Looks like something was not finished here :)

Reply:
oops 🙈